src/sentry/migrations/0167_rm_copied_useroptions.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 1.11.29 on 2021-03-01 20:08
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn"t be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
```

Comment:
Set this to `True`, I think this migration will take a while to run, so better to have ops run it manually (maybe we can ask them to tell us how long it takes so we have an idea for future data migrations.)